draft/index.html

```diff
@@ -124,20 +121,506 @@
+
+<dl>
+<dt>Pflichtfeld</dt>
+<dd>nein</dd>
+<dt>Typ</dd>
+<dd>`object[]`</dd>
+<dt>Werte</dt>
+<dd>JSON-Objekt mit Properties `type` und `name`</dd>
+<dt>Validierung</dt>
+<dd><a href="https://dini-ag-kim.github.io/lrmi-profile/draft/schemas/creator.json">JSON Schema</a></dd>
+</dl>
+
+</section>
+
+<section data-dfn-for="contributor">
+
+### <dfn>contributor</dfn>
+
+Sonstige Beitragende zu der Ressource. MUSS ein oder mehrere JSON-Objekte enthalten, die wiederum `type` und `name` Properties aufweisen MÜSSEN und `id` – mit einer URI-Referenz zu [[?ORCID]], [[?GND]], [[?Wikidata]] oder [[?ROR]] – aufweisen SOLLEN sowie weitere Properties enthalten KÖNNEN, z. B. `email`, `location`, `url`.
+
+<dl>
+<dt>Pflichtfeld</dt>
+<dd>nein</dd>
+<dt>Typ</dd>
+<dd>`object[]`</dd>
+<dt>Werte</dt>
+<dd>JSON-Objekt mit Properties `type` und `name`</dd>
+<dt>Validierung</dt>
+<dd><a href="https://dini-ag-kim.github.io/lrmi-profile/draft/schemas/contributor.json">JSON Schema</a></dd>
+</dl>
+
+</section>
+
+<section data-dfn-for="description">
+
+
+### <dfn>description</dfn>
+
+Beschreibung der Ressource.
+
+<dl>
+<dt>Pflichtfeld</dt>
+<dd>nein</dd>
+<dt>Typ</dt>
+<dd>`string`</dd>
+<dt>Validierung</dt>
+<dd><a href="https://dini-ag-kim.github.io/lrmi-profile/draft/schemas/description.json">JSON Schema</a></dd>
+</dl>
+
+</section>
+
+<section data-dfn-for="about">
+
+### <dfn>about</dfn>
+
+Fach/Thema der Inhalte.
+MUSS einen Wert aus der Destatis Fächerklassifikation (https://w3id.org/kim/hochschulfaechersystematik/scheme) ODER aus der OpenEduHub-Fächersystematik für den Schulbereich (http://w3id.org/openeduhub/vocabs/discipline) enthalten.
+
+<dl>
+<dt>Pflichtfeld</dt>
+<dd>nein</dd>
+<dt>Typ</dt>
+<dd>`string[]` (URI)</dd>
+<dt>Werte</dt>
+<dd>Werte aus <a href="https://w3id.org/kim/hochschulfaechersystematik/scheme">Destatis Fächerklassifikation</a> oder <a href= "http://w3id.org/openeduhub/vocabs/discipline">OpenEduHub-Fächersystematik</a></dd>
+<dt>Validierung</dt>
+<dd><a href="https://dini-ag-kim.github.io/lrmi-profile/draft/schemas/about.json">JSON Schema</a></dd>
+</dl>
+
+</section>
+
+<section data-dfn-for="keywords">
+
+### <dfn>keywords</dfn>
+
+Schlagworte zur Beschreibung des Inhalts der Ressource.
+
+<dl>
+<dt>Pflichtfeld</dt>
+<dd>nein</dd>
+<dt>Typ</dt>
+<dd>`string[]`</dd>
+<dt>Validierung</dt>
+<dd><a href="https://dini-ag-kim.github.io/lrmi-profile/draft/schemas/keywords.json">JSON Schema</a></dd>
+</dl>
+
+</section>
+
+<section data-dfn-for="license">
+
+### <dfn>license</dfn>
+
+Lizenz der Ressource. Wert MUSS eine URI der entsprechenden CC-Lizenz sein.
+
+<dl>
+<dt>Pflichtfeld</dt>
+<dd>nein</dd>
+<dt>Typ</dt>
+<dd>`string[]` (URL entsprechend dem RegEx-Muster `^https:\/\/creativecommons.org\/(licenses|licences|publicdomain)\/.*`)</dd>
```

Comment:
Form der Lizenz sollte vllt. nicht in Form des Regex angegeben werden. Vllt. nur URI/URL der CC-Lizenz.